.vscode/settings.template.json

```
@@ -79,6 +79,7 @@
 	"cmake.generator": "Ninja",
 	"cmake.configureSettings": {
 		"TARGET_BOARD": "LEKA_V1_2_DEV",
+		"OS_VERSION": "1.2.3",
```

To avoid any confusion
```suggestion
		"OS_VERSION": "0.0.0",
```